CMakeLists.txt

```diff
@@ -104,7 +105,7 @@ set(LICENSE "Mozilla Public License Version 2.0")
 
 # Build, project and include settings
 find_package(Threads REQUIRED)
-if(WITH_DLT_COREDUMPHANDLER OR WITH_DLT_FILETRANSFER)
+if(WITH_DLT_COREDUMPHANDLER OR WITH_DLT_FILETRANSFER OR WITH_DLT_LOGSTORAGE_GZIP)
     set(ZLIB_LIBRARY "-lz")
     find_package(ZLIB REQUIRED)
```

Comment:
Unless gzwrite is being deprecated I see no value in alternating between the two. If that is the case we could just stick to gzwrite.

I think we could check the zlib version to prevent compiler errors but mixing around the code feels overkill.

Reply:
Hello @LiquidityC , hello @lvklevankhanh ,
we already decided that we only want to support the new zlib version, so we can stick to gzfwrite().
I think it makes sense to add the minimum version number to the find_package check:
find_package(ZLIB 1.2.9 REQUIRED)

Reply:
Agree. I'll send a patch shortly.

Reply:
Agree with @michael-methner's comment.